### ISSUE TITLE
How can i use "\n" in my string?

### ISSUE THREAD

**Issue by MonjeambYannick** [rougier/freetype-gl#206]
I want to use \n to display multi line text, but it only shows a white rectangle instead of creating a new line.

**Comment by rougier** [comment]
Yes, escape codes are not interpreted. But there are several examples showing how to do it.

**Comment by KironDevCoder** [comment]
You can write some code to handle multiline text @MonjeambYannick.
